release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -201,6 +201,19 @@ Previously, quick start prerequisites were displayed as combined text instead of
 
 image::quick-start-prerequisites.png[quick start prerequisites displayed as popover,338,336]
 
+[id="ocp-4-8-web-console-developer-perspective"]
+==== Developer perspective
+
+* As a cluster administrator, to create and deploy an application from a Git repository, you can use custom pipeline templates that override the default pipeline templates provided by {pipelines-title} 1.5 and later.
```

Comment:
In your other bullets, you don't lead with the phrase, `As a X...` so for consistency, let's not do that here. 

What do you think about this rewrite?

`Use custom pipeline templates to create and deploy an application from a Git repository. These templates override the default pipeline templates provided by {pipelines-title} 1.5 and later.` 

Also, I'm not quite sure why there's `{pipelines-title}`?

Reply:
Done and fixed!